test/common/index.js

```diff
@@ -24,6 +24,8 @@
 const process = global.process;  // Some tests tamper with the process global.
 const path = require('path');
 const fs = require('fs');
+/* eslint-disable no-restricted-properties */
```

Comment:
Please use `// eslint-disable-next-line no-restricted-properties` so it only is disabled for the one line that needs it and not the entire remainder of the file. Thanks!

Although it may not be necessary at all depending on next comment....

Reply:
Er, rather, the point is can you move the rule to be next to where it's needed and only have it apply to that line? Doesn't look like it applies to the next line at all which is kind of confusing... or maybe I'm just really confused for some other reason...

Reply:
It seems like this eslint-disable comment is completely unnecessary. Please remove it.